site/content/entry/yuzu-progress-report-may-2022/index.md

```diff
@@ -0,0 +1,235 @@
+A thread that acquires the lock of mutex can use the critical section while other threads must wait till the lock is released.
+
+{{< imgs
+    "./mutex.png| Example of a mutex"
+  >}}
+
+In theory, when a thread tries to lock a mutex and it does not succeed, because the mutex is already locked, it will go to sleep, immediately allowing another thread to run. 
+It will continue to sleep until being woken up, which will be the case once the mutex is being unlocked by whatever thread was holding the lock before.
+
+But when a thread tries to lock a spinlock and it does not succeed, it will continuously re-try locking it, until it finally succeeds; thus it will not allow another thread to take its place until the OS forces it to timeout.
+Hence, polling on a spinlock will constantly waste CPU time and if the lock is held for a longer amount of time, this will waste a lot more CPU time and it would have been much better if the thread was sleeping instead.
+
+However in practice, most modern operating systems (Windows/Linux) use hybrid mutexes and hybrid spinlocks.
+So spinlocks are fine on systems with tons of cores, as it's not a big deal for host threads to spin. 
+On low core counts, you really start starving the system by doing that, whereas a mutex will allow another thread to be scheduled on that core/thread.
+
+Thus {{< gh-hovercard "8172" "by moving from spinlocks to mutexes," >}} we were able to improve how yuzu runs on systems with low core counts. 
+Our testing results showed that yuzu is now much more usable on 4 thread systems.
+
+## UI changes
+
+[Docteh](https://github.com/Docteh) has been very helpful with some translation holes we had for a while.
+
+For example, the Custom RTC setting had several issues if the Windows system locale was set in certain languages, making it either display incorrectly (for example lacking the AM/PM indicator), or completely unusable.
+{{< gh-hovercard "8291" "Fixing the display format" >}} allows Custom RTC to show up correctly in any language now.
+
+The Network tab in `Emulation > Configure… > System` could remain untranslated after changing languages. This was a simple case of forgetting to include the tab in the translations, so Docteh {{< gh-hovercard "8293" "fixed the oopsie" >}} and the lone Network tab now displays as it should.
+
+{{< single-title-imgs-compare
+    "Netto-kun. Now I want a Battle Network Legacy Collection, c'mon CAPCOM!"
+    "./netbug.png"
+    "./netfix.png"
+>}}
+
+yuzu’s icon in the About dialog has been in a struggle lately, a fix for Linux builds would break Windows support, and vice versa.
+Via qtcreator, Docteh {{< gh-hovercard "8339" "fixed the About dialog UI file," >}} and removed an old warning caused by the original .png image.
+Thanks for taking the time to properly address the issue once and for all!
+
+## Controller changes
+
+Motion continued reporting data when disabled, causing `Pokémon Let’s Go, Eevee/Pikachu!` to spam `StopSixAxisSensor` errors in the log.
```

Comment:
Segway segway segway -- did Flam even ask for this before? <3

"Onward to the land of german77, where he continues his unrelenting quest to deliver the best user input experience possible in **any emulator ever**!"